pkg/minikube/openshiftversions/openshift_versions.go

```diff
@@ -17,48 +17,125 @@ limitations under the License.
 package openshiftversions
 
 import (
+	"encoding/json"
 	"fmt"
 	"io"
-
-	"github.com/google/go-github/github"
-	"github.com/minishift/minishift/pkg/util"
-	githubutil "github.com/minishift/minishift/pkg/util/github"
-	"github.com/pkg/errors"
+	"net/http"
+	"strings"
+	"time"
 )
 
-const githubOwner = "openshift"
-const githubRepo = "origin"
+type ImageTags struct {
```

Comment:
So why all these structs, if you just want the version? One does not have to unmarshal into a struct. You can just have a map and retrieve the data you are after.